source/plugin/event/causes.rst

```diff
@@ -25,8 +36,27 @@ creating a multitude of subevents for the different source conditions this infor
 Every event provides a ``Cause`` object which can be interrogated for the information pertaining to why the event was
 fired. The Cause object can be retrieved from an event by simply calling :javadoc:`Event#getCause()`.
 
-Retrieving objects from a Cause
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Cause and Context
+~~~~~~~~~~~~~~~~~
+
+The ``Cause`` object contains two distinct sets of information, the cause stack and the :javadoc:`EventContext`.
+
+* The cause stack of the event are the direct causes captured in order of importance. There are no names attached
+  to the objects in the cause stack.
+* The event context contains extra information surrounding the event. Contexts are attached to keys but have no
+  order, they are all equally important.
+
+As an example, if a sheep owned by a player eats some grass, the most direct cause of this is the sheep. The
+player would be in the ``EventContext`` as the :javadoc:`EventContextKeys#OWNER`, giving event consumers
+that additional information about how the event has come about, but would not necessarily be within the
+direct cause itself.
+
+Another example that you may need to watch out for is if you simulate a player. The simulated player may not be
+in the direct cause, as the player being simulated may not have been involved in the action, however, the player's
+be in the ``EventContext`` under the :javadoc:`EventContextKeys#PLAYER_SIMULATED`
```

Comment:
The syntax here is funny peculiar. "the player's be in the" sounds like Pirate to me ...